django_jalali/db/models.py

```diff
@@ -318,8 +318,7 @@ def value_to_string(self, obj):
         if val is None:
             data = ''
         else:
-            d = datetime_safe.new_datetime(val)
-            data = d.strftime('%Y-%m-%d %H:%M:%S')
+            data = str(val)
```

Comment:
@itmard value is `jdatetime` type and formatting done in that class.
for compatibility i will use `smart_text` util of django.